library/extractor/src/main/java/com/google/android/exoplayer2/extractor/mp4/AtomParsers.java

```diff
@@ -1173,6 +1173,17 @@ private static void parseVideoSampleEntry(
       } else if (childAtomType == Atom.TYPE_vpcC) {
         ExtractorUtil.checkContainerInput(mimeType == null, /* message= */ null);
         mimeType = (atomType == Atom.TYPE_vp08) ? MimeTypes.VIDEO_VP8 : MimeTypes.VIDEO_VP9;
+        parent.setPosition(childStartPosition + Atom.HEADER_SIZE);
+        // vpcC atom parsing based on FFmpeg implementation
+        // see https://github.com/FFmpeg/FFmpeg/blob/master/libavformat/vpcc.c
```

Comment:
Here is the link to the generic VP9/MP4 spec: https://www.webmproject.org/vp9/mp4/ See: VP Codec Configuration Box -> Syntax. May be we should replace ffmpeg specific implementation link.

Reply:
Thanks for the link! I've replaced the link and updated comments to explain which fields are skipped here. Please take another look.